packages/ui-app/src/Voting.tsx

```diff
@@ -36,25 +36,25 @@ class Voting extends TxModal<Props, State> {
   state: State;
 
   headerText = () => {
-    const { isCouncil, t } = this.props;
+    const { isCollective, t } = this.props;
 
-    return t(isCouncil ? 'Vote on council motion' : 'Vote on proposal');
+    return t(isCollective ? 'Vote on collective proposal' : 'Vote on proposal');
   }
 
   accountLabel = () => this.props.t('Vote with account');
   accountHelp = () => this.props.t('Select the account you wish to vote with. You can approve "aye" or deny "nay" the proposal.');
 
   txMethod = () => {
-    const { isCouncil } = this.props;
+    const { isCollective } = this.props;
 
-    return isCouncil ? 'councilMotions.vote' : 'democracy.vote';
+    return isCollective ? 'collective.vote' : 'democracy.vote';
```

Comment:
So my fear here is that as we push this, Alex is not working. Which I think is ok.